src/vmm/src/vstate.rs

```diff
@@ -1310,14 +1310,17 @@ mod tests {
     use std::os::unix::io::AsRawFd;
     #[cfg(target_arch = "x86_64")]
     use std::path::PathBuf;
-    use std::sync::{mpsc, Arc, Barrier};
+    #[cfg(target_arch = "x86_64")]
+    use std::sync::mpsc;
+    use std::sync::{Arc, Barrier};
     #[cfg(target_arch = "x86_64")]
     use std::time::Duration;
 
     use super::super::devices;
     use super::*;
 
     use utils::signal::validate_signal_num;
+    #[cfg(target_arch = "x86_64")]
     use vmm_config::boot_source::DEFAULT_KERNEL_CMDLINE;
```

Comment:
nit: or you can move this in the x86_64 specific function that uses it (i.e load_good_kernel)